mkdocs.yml

```diff
@@ -386,6 +390,8 @@ nav:
               - 'pfSense (serial output) booting performance test': unified-test-documentation/dasharo-performance/414-pfsense-serial-booting-performance-test.md
               - 'pfSense (VGA output) booting performance test': unified-test-documentation/dasharo-performance/415-pfsense-vga-booting-performance-test.md
               - 'Windows booting performance test': unified-test-documentation/dasharo-performance/416-windows-booting-performance-test.md
+          - 'Dasharo stability':
+              - 'NET interface check after coldboot/warmboot/reboot/suspend': unified-test-documentation/dasharo-stability/01-net-controller-after-coldboot-warmboot-reboot-suspend.md
```

Comment:
This change is irrelevant, please revert.

Reply:
I reverted the change.